<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addgroup</class>
 <widget class="QWidget" name="addgroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>创建群聊</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>361</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>361</width>
     <height>85</height>
    </rect>
   </property>
   <property name="accessibleDescription">
    <string>验证信息</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
    border: 1px solid gray;
    border-radius: 5px;
    padding: 2px 10px;
    font-size: 16px;
    color: black;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_create">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>390</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>创建</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
    border: 1px solid gray;
    border-radius: 5px;
    padding: 2px 10px;
    font-size: 16px;
    color: black;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>380</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
